cloudgenix_config/do.py

```diff
@@ -3875,8 +3896,13 @@ def get_pppoe_id(config_pppoe_interface, interfaces_cache, interfaces_n2id, conf
     :return: Matching PPPoE Interface ID
     """
     return_if_id = None
-    parent_if_id = interfaces_n2id.get(config_pppoe_interface.get('parent', ""))
     parent_if_name = config_pppoe_interface.get('parent', "")
+    # Below check is for 9k. Bypasspairs in range 12-16 is saved with an '_' for easy identification
+    if config_pppoe_interface.get('parent_type', None) == 'bypasspair' and int(parent_if_name) in range(12, 17):
+        parent_if_id = interfaces_n2id.get(config_pppoe_interface.get('parent', "") + '_bypasspair', parent_if_name)
+    else:
+        parent_if_id = interfaces_n2id.get(config_pppoe_interface.get('parent', ""), parent_if_name)
+    parent_if_id = interfaces_n2id.get(config_pppoe_interface.get('parent', ""))
```

Comment:
why are we recalculating parent_if_id? Looks like a typo.

Reply:
Fixed this typo